torchao/prototype/moe_training/kernels/mxfp8_blocked_scales.py

```diff
@@ -216,14 +240,14 @@ def triton_mx_block_rearrange_2d_M_groups(
         "Expected element size to be 1 byte (8 bits)"
     )
     rows, cols = scales_tensor.shape
-    num_groups = input_group_end_offsets.numel()
+    num_groups = input_group_end_offsets.shape[0]
 
     # Final offset is the total number of rows in the tensor
-    padded_rows = output_group_start_offsets[-1]
+    padded_rows = rows + num_groups * 128
```

Comment:
can you write a `note: padding to max_possible: ...`
and then link to that note throughout the codebase this is a pretty common pattern